Source/WebCore/dom/Node.cpp

```diff
@@ -2297,7 +2297,7 @@ void Node::moveTreeToNewScope(Node& root, TreeScope& oldScope, TreeScope& newSco
             if (newScopeIsUAShadowTree)
                 node.setEventTargetFlag(EventTargetFlag::HasBeenInUserAgentShadowTree);
             node.setTreeScope(newScope);
-            if (UNLIKELY(!node.hasRareData()))
+            if (!node.hasRareData()) [[unlikely]]
```

Comment:
Wait a minute. This unlikely is wrong. It should be `[[likely]]` instead since we expect not having rare data to be the common case.